docs/add-agent.md

```diff
@@ -183,6 +183,31 @@ Make sure you've read the [Actuated EULA](https://github.com/self-actuated/actua
 
     You can also run the Actuated Agent software as a systemd unit file for automatic restarts and to start upon boot-up.
 
+    An example of a systemd file you can use:
+
+    ```systemd
+    [Unit]
+    Description=Actuated Agent
+    [Service]
+    User=<user>
+    Type=simple
+    ExecStart=<user>/start.sh
+    Restart=always
+    RestartSec=5s
+
+    [Install]
+    WantedBy=multi-user.target
+    ```
+
+    Check the `User` and `ExecStart` directives and make sure to replace `<user>` with the name of the user account for the agent. The usual place to save the unit file is: `/etc/systemd/system/actuated-agent.service`.
+
+    After saving your service file, you can start the service for the first time.
+
+    ```bash
+    sudo systemctl daemon-reload
+    sudo systemctl enable --now actuated-agent
```

Comment:
Shorter and easier to type if it's called `actuated` I think?

Reply:
The service has been renamed to `actuated`.